src/app/weak-supervision/components/crowd-labeler-details/component/crowd-labeler-details.component.html

```diff
@@ -187,7 +187,7 @@ <h3 class="mt-2 text-3xl font-bold leading-8 tracking-tight text-gray-900 sm:tex
                                 valuePropertyPath:'id',
                                 isDisabled:annotators.length == 0,
                                 isDisabled:informationSource?.lastTask ? informationSource?.lastTask.progress !=0:false
-                            }" (optionClicked)="changeSettings('annotatorId',$event)">
+                            }" (optionClicked)="changeSettings('annotatorId',$event, false)">
```

Comment:
why were these set to false? This results in changes in the annotator id not being saved to the database?

Reply:
This was set to false because it was updating the information source and once both dropdowns were updated the link was generated

Reply:
this doesn't work since the values aren't saved as needed